presto-docs/src/main/sphinx/release/release-0.252.rst

```diff
@@ -0,0 +1,40 @@
+=============
+Release 0.252
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix hanging join operator when spill is enabled and the probe side finishes before the hash builder starts.
+* Add file based password authenticator plugin.
+* Add new spilling strategy to spill when the user + system + revocable memory on a node exceeds max_total_memory_per_node.  This can be enabled by setting ``experimental.spiller.task-spilling-strategy`` to ``PER_QUERY_MEMORY_LIMIT``.
+* Add support for logging formatted prepared query. This could be enabled by setting session property ``log_formatted_query_enabled`` to ``true``.
+* Add support for passthrough Elasticsearch queries.
```

Comment:
Move to elasticsearch connector

Reply:
We should frame this better - 

```
Add support for running Presto queries over the results of Elasticsearch queries. See :doc:`connector/elasticsearch` on how to use it
```

@aweisberg to confirm if I rephrased this correctly

Reply:
Yes sort of. We already supported it. The difference is these are passthrough in that you don't need to escape the query contents because they are base32 encoded.

Maybe
``` 
Add support for running base32 encoding ElasticSearch queries that are passed through. See :doc:`connector/elasticsearch` on how to use it
```